### PR TITLE
chore: update branch name to main where its used

### PR DESCRIPTION
**Pull request by vdovhanych**

### CODE REVIEW

**Review by matejcik — APPROVED** [review]
utACK but let's see if CI is happy first

File: ci/posttest.yml [hunk 1]
```diff
@@ -43,10 +43,10 @@ unix ui changes:
     - core device R test
     - legacy device test
   script:
-    - $NIX_SHELL --run "poetry run python -m tests.ui_tests.reporting master-diff"
-    - mv tests/ui_tests/reports/master_diff .
+    - $NIX_SHELL --run "poetry run python -m tests.ui_tests.reporting main-diff"
+    - mv tests/ui_tests/reports/main_diff .
```
Comment (matejcik) [inline ci/posttest.yml:47]: the command is still called `master-diff` and the path is hardcoded to `master_diff` it seems.
honestly i'd maybe keep those names? it seems more readable than "main_diff", which sounds more like "the main one of all the various diffs" than "diff from main branch".
(or rename it to "diff-from-main")

Comment (vdovhanych) [reply ci/posttest.yml]: O, I'll check it.

Comment (matejcik) [reply ci/posttest.yml]: f93d9a636d93370dfa68ecb49d1eb7dd8b236627

**Review by vdovhanych — COMMENTED** [review]
lgtm then :)

**Pull request merged**